src/hotspot/share/logging/logFileStreamOutput.cpp

```diff
@@ -32,8 +32,6 @@
 #include "utilities/defaultStream.hpp"
 
 const char* const LogFileStreamOutput::FoldMultilinesOptionKey = "foldmultilines";
-LogStdoutOutput* StdoutLog = nullptr;
-LogStderrOutput* StderrLog = nullptr;
```

Comment:
Surely these had to be declared in a header file somewhere, but I don't see it being removed from a header file,

Reply:
Ouch, good catch, thank you.